src/store/index.js

```diff
@@ -5,19 +5,30 @@ import { INITIALIZE_PNP } from './action-types.js'
 import { UPDATE_AVAILABLE } from './mutation-types'
 
 Vue.use(Vuex)
+let edgeVersion = require('@/../package.json').version
+
+fetch('http://localhost:8778/api/status')
+  .then((res) => res.json())
+  .then((response) => {
+    if (response.version && response.status === 'OK') {
+      edgeVersion = response.version
+    }
+  })
+  .catch((e) => {
+    console.log(`Error getting version: ${e}`)
+  })
 
 const store = new Vuex.Store({
   state: {
     updateToBeInstalled: undefined,
-    version: require('@/../package.json').version
+    edgeVersion
```

Comment:
@vickywane this is a rushed patch.

We still need to show the PWA version in the About page. Separately from it, we want to show the Edge version in the edge card under settings.

Please add at least two new test cases to prevent future regression like this change:

1. Check that PWA version is correctly displayed in the About page.
2. Check that Edge version is correctly displayed in the Edge Settings Card.